cli/command/secret/ls.go

```diff
@@ -44,25 +42,13 @@ func runSecretList(dockerCli *command.DockerCli, opts listOptions) error {
 	if err != nil {
 		return err
 	}
-
-	w := tabwriter.NewWriter(dockerCli.Out(), 20, 1, 3, ' ', 0)
-	if opts.quiet {
-		for _, s := range secrets {
-			fmt.Fprintf(w, "%s\n", s.ID)
-		}
-	} else {
-		fmt.Fprintf(w, "ID\tNAME\tCREATED\tUPDATED")
-		fmt.Fprintf(w, "\n")
-
-		for _, s := range secrets {
-			created := units.HumanDuration(time.Now().UTC().Sub(s.Meta.CreatedAt)) + " ago"
-			updated := units.HumanDuration(time.Now().UTC().Sub(s.Meta.UpdatedAt)) + " ago"
-
-			fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", s.ID, s.Spec.Annotations.Name, created, updated)
-		}
+	format := opts.format
+	if len(format) == 0 {
```

Comment:
And add an example to the CLI docs here; https://github.com/docker/docker/blob/master/docs/reference/commandline/cli.md#configuration-files